app/views/spaces/edit.html.erb

```diff
@@ -24,7 +24,7 @@
         </li>
       <% end %>
 
-      <% if space.entrance.furnitures.blank? %>
+      <% if space.entrance&.furnitures.blank? %>
```

Comment:
I *think* nil respods to `blank?` so it should be fine; but in general once you safe-navigate you'll want to safe-navigate the rest: `space.entrance&.furnitures&.blank?`

Reply:
ah i forgot. thanks! added c61aa9db7578961cf8c734bd697be56c9f34b4da